src/OsiCbcSolverInterface.cpp

```diff
@@ -7,6 +7,8 @@
 #include "OsiCbcSolverInterface_2_8.cpp"
 #elif CBC_VERSION_MAJOR == 2 && CBC_VERSION_MINOR == 9
 #include "OsiCbcSolverInterface_2_9.cpp"
+#elif CBC_VERSION_MAJOR == 2 && CBC_VERSION_MINOR == 10
```

Comment:
I know this pattern was inherited from before, but why not just include the `hpp` file here, instead?